EinvitationMaker/src/components/EditingPage/EditingPage.js

```diff
@@ -233,6 +203,37 @@ function EditingPage() {
   //       .catch((err) => console.log(err));
   //   }
   // };
+  
+
+  const handleDownload = () => {
+    html2canvas(templateRef.current, {
+      logging: true,
+      scrollY: -window.scrollY,
+    }).then((canvas) => {
+      canvas.toBlob((blob) => {
+        saveAs(blob, "template.png");
+      });
+    });
+
+    if (isAuthenticated) {
+      fetch("http://localhost:4000/editing", {
```

Comment:
Add `http://localhost:4000` to `.env` and use `process.env` here, later when you change the domain while you host it you only have to change one line.

Reply:
Done.